src/components/Global/ValidatedInput/index.tsx

```
@@ -107,9 +109,12 @@ const ValidatedInput = ({
                     type="text"
                     value={formatDisplayValue ? formatDisplayValue(value) : value}
                     onChange={handleChange}
-                    className="h-12 w-full bg-white pl-4 pr-2 text-h8 font-medium 
-                        outline-none focus:outline-none active:bg-white
-                        dark:bg-n-1 dark:text-white dark:placeholder:text-white/75"
+                    className={twMerge(
+                        `h-12 w-full border-0 bg-white pr-2 text-h8 
+                        font-medium outline-none focus:outline-none
+                        active:bg-white dark:bg-n-1 dark:text-white dark:placeholder:text-white/75`,
+                        !!infoText ? 'pl-0' : 'pl-4'
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)